Database/Corrections/sodQuestFixes.lua

```diff
@@ -3164,10 +3164,70 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredClasses] = classIDs.SHAMAN,
-            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receieve the rune."},
+            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receive the rune."},
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90209] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{414624}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Lighthouse Stash under the right of the tree to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90210] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{407731}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Stonemason's Toolbox on the platform behind the quest givers at Stonewrough Dam to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90211] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {{210549}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 15,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Approach the Defias Scout in Stealth and Pickpocket them to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90212] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{407457}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Take a left from the main gate, go up and around to the right and jump down onto the wall behind the stable and drop on to the roof and open the Stable Hand's Trunk to receive the rune. NOTE: You need 80 Lockpicking to open the trunk."},
```

Comment:
Technically we can add the `requiredSkill` field to show this rune not before the player reached Lockpicking 80. I am not sure if we should though 🤔

Reply:
Looking over it, that should be possible, but we would need to add `Lockpicking` as a skill to `QuestieProfessions.lua` and then use `requiredSpecialization` to set it to Lockpicking 80.

Reply:
Well yeah, I know that it is possible. I am honestly unsure if we should do it though.